dev/ci/ci-user-overlay.sh

```diff
@@ -29,3 +30,8 @@ if [ $TRAVIS_PULL_REQUEST == "461" ] || [ $TRAVIS_BRANCH == "stm+remove_compat_p
     mathcomp_CI_BRANCH=no_camlp4_compat
     mathcomp_CI_GITURL=https://github.com/ejgallego/math-comp.git
 fi
+
+if [ $TRAVIS_PULL_REQUEST == "565" ] || [ $TRAVIS_BRANCH == "remove-vernacerror" ]; then
+    mathcomp_CI_BRANCH=remove-vernacerror
+    mathcomp_CI_GITURL=https://github.com/SkySkimmer/math-comp.git
```

Comment:
Although not the currently recommended way, I think that:

    mathcomp_CI_BRANCH=refs/pull/120/head

would work as well. It could be better from a history-keeping point of view (e.g. if you delete your fork or your branch). I don't know what @ejgallego thinks.